### PR TITLE
Simulate all worm controllers, only allow input on IsCurrentTurn, added hats.

### LINKED ISSUE #34: Worms that haven't had an active turn yet aren't behaving properly
Note this is the case for additional worms belonging to a player. The first worm spawned for any given player seems to load in just fine.

### PR DESCRIPTION
**Pull request by Matt9440**
Each players pawn now becomes Pawn.Player rather than a worm. That player then simulates all of it's associated worms and only allows input where the worms IsCurrentTurn bool is set to true.

Fixes #34 
Implements #27 camera

### CODE REVIEW

**Comment by Matt9440** [comment]
Found an issue whereby local worms can still fire after their turns are ended. Will resubmit when fixed.

File: code/Pawn/Camera.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ public void SetLookTarget( Entity target )
 		public override void Update()
 		{
 			var pawn = LookTarget;
+			//Log.Info( pawn );
```
Comment (trundler-dev) [inline code/Pawn/Camera.cs:37]: Missed one 😺

**Review by trundler-dev — APPROVED** [review]
Just get rid of that Log and feel free to merge! Good work.

**Review by xezno — APPROVED** [review]
LGTM

**Pull request merged**